.circleci/config.yml

```diff
@@ -0,0 +1,213 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+
+# CircleCI maintains a library of pre-built images
+# documented at https://circleci.com/docs/2.0/circleci-images/
+
+# Default values
+defaults: &defaults
+  docker:
+    - image: circleci/node:6-browsers
+  working_directory: ~/ember-cli-react
+
+version: 2
+jobs:
+  checkout_code:
+    <<: *defaults
+    steps:
+      - checkout
+      - persist_to_workspace:
+          root: .
+          paths:
+            - .
+
+  install_dependencies:
```

Comment:
Not necessary to change, but having two separate jobs for code checkout and installing dependencies may slow things down slightly. These jobs run in a series and there's some overhead to moving from one job to the next and restoring workspace cache.

Reply:
Oh that's a good suggestion, I'll combine them together